spec/features/bars/user_views_all_bars_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+feature 'user can view a list of bars', %{
+  Acceptance Criteria
+[] User can see a list of all bars sorted most recently created
+[] User does not have to be logged in to see a list of bars
+} do
+
+  scenario 'user visits home page' do
+    bar1 = FactoryGirl.create(:bar)
+    bar2 = FactoryGirl.create(:bar2)
+    bar3 = FactoryGirl.create(:bar3)
```

Comment:
Look into `create_list` in the [docs](https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md):

[SO discussion](http://stackoverflow.com/questions/18445828/factorygirlcreate-the-same-object-multiple-times)